### ISSUE TITLE
Deploy Superset on GCP

### ISSUE THREAD

**Issue by atharmarajah** [apache/incubator-superset#7287]
Hi, I would like to deploy Superset using Google Cloud Platform. I am using amancevice/superset docker images and I follow this tutorial : https://cloud.google.com/kubernetes-engine/docs/tutorials/hello-app

Instead of creating the images with docker build I just commited the containers to get images.

I have been able to do the first steps but when I want to do the step 3 and execute the container of the image locally before deplying it fails. 
Many errors are displayed and I don't get anything on my localhost.

Can I get more indications on how I could deploy it correctly?
Thanks

**Comment by MarcusSorealheis** [comment]
Can you post exactly what you have done and maybe the relevant lines of the console log? I have deployed Superset on GCP with no issue. Maybe I can help.

**Comment by atharmarajah** [comment]
I have been able to progress on this issue. Instead of using amancevice/superset docker image I am using tylerFowler/docker-superset image now. I have been able to follow all the steps and my application has been deployed to an external IP address (http://130.211.228.43). I would like to know how I could modify the domain name to a customized one instead of having an external IP address to access it. Thanks
